docs/entities/Case.md

```diff
@@ -439,18 +427,16 @@
       type: "boolean"
       flags: 
         presence: "optional"
-    leadCaseId: 
+    leadDocketNumber: 
       type: "string"
       flags: 
         presence: "optional"
-        description: "If this case is consolidated, this is the ID of the lead case. It is the lowest docket number in the consolidated group."
+        description: "If this case is consolidated, this is the docket number of the lead case. It is the lowest docket number in the consolidated group."
       rules: 
         - 
-          name: "guid"
+          name: "pattern"
           args: 
-            options: 
-              version: 
-                - "uuidv4"
+            regex: "/^([1-9]\\d{2,4}-\\d{2})$/"
```

Comment:
sweet; makes me wonder whether this will have to change when we have all of the cases in the system.

Reply:
Using a UUID4? From wikipedia: `the probability to find a duplicate within 103 trillion version-4 UUIDs is one in a billion`

Reply:
err, i meant the docket 1234-12
```
 regex: "/^([1-9]\\d{2,4}-\\d{2})$/"
```

Reply:
I think I get it, though, 99999 max cases in a 2-digit year.